assets/prototype/domain/shared/predicates/datetimes/validStatus.ts

```diff
@@ -8,11 +8,7 @@ import { is } from 'ramda';
  */
 import { Datetime } from '../../../eventEditor/data/types';
 
-/**
- * @param {Object} dateEntity    event dateEntity object
- * @return {boolean} true if capacity property is valid
- */
-const validStatus = (date: Datetime) => {
+const validStatus = (date: Datetime): boolean => {
 	return is(String, date.status);
 };
```

Comment:
we could likely use an enum for this, since there are a set number of values (see: `core\db_classes\EE_Datetime.class.php`

Reply:
Updated here: https://github.com/eventespresso/event-espresso-core/pull/2174/commits/1e0f04e427c0b23d74c4c09bd739799e07f7e864

Reply:
@tn3rb had added enums, but due to some issue, didn't added that to `Datetime` type.
Will need to investigate again after this: https://github.com/eventespresso/event-espresso-core/issues/2143